res/controllers/Reloop-Jockey-3-ME-scripts.js

```diff
@@ -0,0 +1,546 @@
+function Jockey3ME() {}
+
+// Variables
+Jockey3ME.EffectLedMeter = 0;
+Jockey3ME.EffectLedMeterValue = 1;
+Jockey3ME.LedMeterShowValue = 1;
+Jockey3ME.LedMeterShowValueTwo = false;
+Jockey3ME.scratching = [];
+Jockey3ME.jogwheelResolution = 2048;
+Jockey3ME.jogwheelSpinSpeed = 33+1/3;
+Jockey3ME.hotcueClearVal = 0;
+Jockey3ME.num_effectsValue = [0,0,0,0];
+Jockey3ME.effectsAvailable = 5; // Sets how many Effects are Loadable
+Jockey3ME.fxSelectKnobPush = [0,0,0,0];
+Jockey3ME.fxSelectKnobPushIterator = [0,0,0,0,0,0,0,0,0,0,0,0];
+Jockey3ME.fxSelectKnobPushLedTemp = 0;
+Jockey3ME.fxSelectKnobParamChose = 0;
+Jockey3ME.fxSelectKnobParamLinkChose = 0;
+Jockey3ME.fxSelectKnobParamLinkInverseChose = 0;
+Jockey3ME.loop_move_value = 4; // Sets how many a Loop Jumping Beats when "MOVE" is Turned
+Jockey3ME.loop_move_bool = false;
+Jockey3ME.MixerDeck1 = 0;
+Jockey3ME.MixerDeck2 = 0;
+
+script.crossfaderCurve = function (value, min, max) {
```

Comment:
Oh, that was easy. I have changed some values on this code and now it works perfectly i think:
`var newValue = script.absoluteLin(value, 2, 150, min, max);`
in line 26 and
`engine.setValue("[Mixer Profile]", "xFaderCalibration", script.absoluteLin(value, 0, newValueThree, min, max));`
in line 30.
I think that acts like expected. What do you think?